docker/entrypoint.sh

```diff
@@ -1,28 +1,85 @@
 #!/bin/bash
 gcloud auth activate-service-account --key-file=/secrets/gcloud_key.json
-#sleep 3600
 export GOOGLE_APPLICATION_CREDENTIALS=/secrets/gcloud_key.json
-export BUCKET=$1
-export BUCKET_LOCAL=$2
-export UBMLEXP_GIT=$3
-export WARM_START_NAME=$4
-export FILELIST_TRAIN=$5
-export FILELIST_VAL=$6
-export CONFIG=$7
-export LOG_PATH=$8
-export CHECKPOINT_PATH=$9
+if [[ -z "$1" ]]; then
+    echo "Enter BUCKET: "
+    read inp
+    export BUCKET=$inp
+else
+    export BUCKET=$1
+fi
+if [[ -z "$2" ]]; then
+    echo "Enter Experiemnts repo to clone: "
+    read inp
+    export UBMLEXP_GIT=$inp
+else
+    export UBMLEXP_GIT=$2
+fi
+if [[ -z "$3" ]]; then
```

Comment:
does this mean than argument 3 always has to be warm start name?  what if I want arguments 2 and 4?

Reply:
Unfortunately not supported in this branch .... I think we could add command line flags to get that